### ISSUE TITLE
Feature request: no_std

### ISSUE THREAD

**Issue by Kezii** [loony-bean/textplots-rs#14]
Would be nice to use this in embedded devices

**Comment by TDHolmes** [comment]
would need to propogate down to `drawile-rs` & `colored` too. Might be tricky. `drawile-rs` looked fine but `colored` looked potentially tricky.

**Comment by loony-bean** [comment]
I'm not sure this lib is useful in `no_std` setting, at least I cannot imaging any use case.. Maybe we should close this one.

**Comment by Kezii** [comment]
> at least I cannot imagine any use case.

The usecase is exactly the same it has for desktops, lol

**Comment by loony-bean** [comment]
Yeah.. but in most cases the device will not have any display, or it will have some really limited one? This is where my thoughts kind of stop

**Comment by TDHolmes** [comment]
I currently have a firmware project streaming data, and then a host side project doing the plotting. If this was no_std, I could directly plot in firmware in the serial console

**Comment by Kezii** [comment]
> Yeah.. but in most cases the device will not have any display, or it will have some really limited one? This is where my thoughts kind of stop

that's the most compelling reason to have a library like this, when you have no other options other than a textual interface
